src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+
+/**
+ * Feedback session submission page.
+ */
+@Component({
+  selector: 'tm-session-submission-page',
+  templateUrl: './session-submission-page.component.html',
+  styleUrls: ['./session-submission-page.component.scss'],
+})
+export class SessionSubmissionPageComponent implements OnInit {
```

Comment:
This page is for both student/instructor to answer feedback session right?

Reply:
Yes it is, the interface is the same.